subcommands/targets/offline-update.go

```diff
@@ -366,6 +375,44 @@ func downloadTufRepo(factory string, target string, tag string, prod bool, wave
 	return nil
 }
 
+func copyRecursive(src, dst string) error {
+	copyFile := func(src, dst string) error {
+		srcFile, err := os.Open(src)
+		if err != nil {
+			return nil
+		}
+		defer srcFile.Close()
+
+		dstFile, err := os.Create(dst)
+		if err != nil {
+			return err
+		}
+		defer dstFile.Close()
+
+		_, err = dstFile.ReadFrom(srcFile)
```

Comment:
I don't have strong opinion, just `io.Copy` seems like would be more flexible.

Reply:
I thought `io.Copy` could replace the entire function, but unfortunately it does not.

No idea what flexibility you mean... a doc says this:
```
If src implements [WriterTo](https://pkg.go.dev/io#WriterTo),
the copy is implemented by calling src.WriteTo(dst).
Otherwise, if dst implements [ReaderFrom](https://pkg.go.dev/io#ReaderFrom),
the copy is implemented by calling dst.ReadFrom(src).
```
So it seems to only add an extra function call.

Reply:
By flexibility I meant exactly what the doc extract says - it agnostic to details of src and dst implementation. Their type change won't need to change io.Copy() call.
As I said, I don't have strong opinion, just looks more uniform to use `io.Copy`.